src/getTime.js

```diff
@@ -18,7 +18,13 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const validTime = str.match(/\b([0-1][0-9]|[2][0-3]):[0-5][0-9]\b/);
+
+  if (validTime === null) {
+    return '';
+  }
+
+  return validTime[0];
```

Comment:
you can write like this
```suggestion
  return validTime ? validTime[0] : '';
```